src/dynarec/arm64/dynarec_arm64_helper.c

```diff
@@ -974,6 +974,16 @@ void x87_do_pop(dynarec_arm_t* dyn, int ninst, int s1)
         }
 }
 
+int x87_cachecount(dynarec_arm_t* dyn, int ninst)
```

Comment:
Ah, I see, we already have `dyn->n.stack` for this, so this function is unnecessary then...